app/Services/ImportReport.php

```diff
@@ -36,6 +36,7 @@ public function generateCSV(string $path): void
                 $each_import->status,
                 $mismatches_count,
                 $mismatches_wikidata,
+                $mismatches_missing,
```

Comment:
Yeah, I think not including this change in the PR would’ve been more cumbersome, since the header row includes all the accepted statuses (I found this file via a failing test).

Reply:
makes sense :+1: